### PR TITLE
Prevent duplicate names when using extension 100 with SetRequest on gNMI NB

### PR DESCRIPTION
**Pull request by SeanCondon**

### CODE REVIEW

File: tools/test/devicesim/gnmi_target/defs.go [hunk 1]
```diff
@@ -17,7 +17,11 @@ package main
 import (
 	"flag"
 
+<<<<<<< HEAD
```
Comment (tomikazi) [inline tools/test/devicesim/gnmi_target/defs.go]: I assume this should not be part of the patch, correct?

File: tools/test/devicesim/gnmi_target/set.go [hunk 2]
```diff
@@ -31,8 +31,12 @@ func (s *server) Set(ctx context.Context, req *pb.SetRequest) (*pb.SetResponse,
 		return nil, status.Error(codes.PermissionDenied, msg)
 	}
 	log.Infof("allowed a Set request: %v", msg)
+<<<<<<< HEAD
```
Comment (tomikazi) [inline tools/test/devicesim/gnmi_target/set.go]: Same

File: tools/test/devicesim/gnmi_target/defs.go (hunk 1 above)
Comment (SeanCondon) [reply tools/test/devicesim/gnmi_target/defs.go]: something screwed up in my fork

**Comment by SeanCondon** [comment]
will push again later
